### PR TITLE
Add borough as a possible layer for geonames

### PR DESCRIPTION
**Pull request by avulfson17**
Fixes pelias/geonames#99

### CODE REVIEW

**Comment by orangejulius** [comment]
What, no tests? :)

**Comment by avulfson17** [comment]
I think there is a test for it 
`test('alias layer mapping', function(t) {
    t.deepEquals(type_mapping.layer_mapping.coarse,
                 [ 'continent', 'country', 'dependency', 'macroregion',
                   'region', 'locality', 'localadmin', 'macrocounty', 'county', 'macrohood',
                   'borough', 'neighbourhood', 'microhood', 'disputed' ]);
    t.end();
  });`

**Comment by orangejulius** [comment]
Well, there's only one file changed, and it's the implementation code. The test didn't fail in the past, so it couldn't have covered the case you added :P

The test you copied is for the `coarse` alias. I think you want to add another test like this one for `borough`: https://github.com/pelias/api/blob/master/test/unit/sanitiser/_sources_and_layers.js#L54-L63

File: test/unit/sanitiser/_sources_and_layers.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ module.exports.tests.no_errors = function(test, common) {
 
 test('valid combination', function(t) {
     var raw = {};
-    var clean = { sources: ['geonames'], layers: ['macroregion'] };
+    var clean = { sources: ['geonames'], layers: ['borough'] };
```
Comment (orangejulius) [inline test/unit/sanitiser/_sources_and_layers.js]: please duplicate this test, keeping the macroregion one, and adding a new borough one :P

**Comment by dianashk** [comment]
LGTM

**Pull request merged**